api/enki.py

```diff
@@ -368,7 +377,10 @@ def fulfill(self, patron, pin, licensepool, internal_format, **kwargs):
         :return: a FulfillmentInfo object.
         """
         book_id = licensepool.identifier.identifier
-        response = self.loan_request(patron.authorization_identifier, pin, book_id)
+        enki_library_id = self.enki_library_id(patron.library)
+        response = self.loan_request(
+            patron.authorization_identifier, pin, book_id, enki_library_id
+        )
```

Comment:
Library ID also matters here since fulfill and checkout are basically the same operation.